less/components/organize.less

```diff
@@ -0,0 +1,189 @@
+@import "../variables";
+
+@organize-node-bg: #DFE6ED;
+@organize-node-fg: #4B5C6B;
+@organize-branches: #788896;
+
+.organize-tree {
+    margin-bottom: 14px;
+    padding: 12px;
+
+    .top-level-list {
+        padding-inline-start: 0;
+    }
+
+    .list {
+        padding-inline-start: 26px;
+
+        & > .node-container:before {
+            content: "\a0";
+            padding: 0 4px;
+            position: absolute;
+            border-left: 2px dashed @organize-branches;
+            border-bottom: 2px dashed @organize-branches;
+            left: -14px;
+            height: 25.5px;
+        }
+
+        & > .node-container.node-middle:after {
+            content: "\a0";
+            padding: 0 4px;
+            position: absolute;
+            border-left: 2px dashed @organize-branches;
+            left: -14px;
+            bottom: 0;
+            height: ~"calc(100% - 25.5px)";
+        }
+    }
+
+    .node-container {
+        display: flex;
+        flex-direction: column;
+        position: relative;
+
+        &.node-fake .node {
+            opacity: 0.65;
+            border: 2px dashed #788896;
+            padding: 2px;
+        }
+
+        &.node-hide .node {
+            opacity: 0.65;
+            border: 2px dashed #788896;
+            padding: 2px;
+            color: transparent;
+            background-color: transparent;
+
+            .icon {
+                fill: transparent;
+            }
+
+            input {
+                visibility: hidden;
+            }
+        }
+
+        &.node-empty > .node-parent > .node {
+            background-color: #ffc422;
+            border: 2px dashed #ff5722;
+            padding: 2px;
+        }
+    }
+
+    .node-parent {
+        display: flex;
+    }
+
+    .node {
+        display: flex;
+        align-items: center;
+        cursor: pointer;
+        padding: 4px;
+        color: @organize-node-fg;
+        background-color: @organize-node-bg;
+        margin-top: 10px;
+        height: 31px;
+        max-width: 100%;
+
+        & * {
+            // Stop children from spamming dragenter and dragleave events.
+            // https://stackoverflow.com/a/18582960
+            pointer-events: none;
+        }
+
+        & > a {
+            pointer-events: auto;
+
+            display: flex;
+            align-items: center;
+            justify-content: center;
+
+            padding: 8px 6px;
+        }
+
+        & > input {
+            pointer-events: auto;
+
+            margin-right: 4px;
+
+            &, &::placeholder {
+                height: 30px;
+                font-size: 14px;
+                line-height: 1.5;
+            }
+        }
+
+        .icon {
+            fill: @organize-node-fg;
+            margin: 0;
+        }
+
+        .icon.symbol-icon {
+            margin-right: 12px;
+        }
+
+        .text {
+            font-size: 1.2em;
+            margin-right: 6px;
```

Comment:
try to use `@spacer` for margins and padding where possible, to keep things uniform / easier to tweak centrally. It's not always practical (borders, 1 or 2px often makes more sense).